subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/metrics/TestRunnerMetricsListener.kt

```diff
@@ -133,11 +133,9 @@ internal class TestRunnerMetricsListener(
             )
 
             sendTotalTime(provider.totalTime())
-
-            provider.medianDeviceUtilization().fold(
-                { sendMedianDeviceUtilization(it) },
-                { logger.warn("Not sending median device relative wasted time, no data") }
-            )
+            sendDevicesLiving(provider.devicesLiving())
```

Comment:
So, why did I decide to send 3 different indicators comparing to `device-utilization`?

1. Indicators are absolute. So, if we sum series for all tasks we will get `device-utilization` for all tasks
2. We could count the `idle-time-budget`. The amount of time when our devices do nothing across all tasks and devices